jdaviz/core/template_mixin.py

```diff
@@ -101,6 +101,13 @@ def _mxn_update_plugin_opened(self, new_value):
         tray_names_open = [app_state.tray_items[i]['name'] for i in app_state.tray_items_open]
         self.plugin_opened = app_state.drawer and self._registry_name in tray_names_open
 
+    def open_in_tray(self):
```

Comment:
Need a docstring since its API doc is rendered at https://jdaviz--1559.org.readthedocs.build/en/1559/api/jdaviz.core.template_mixin.PluginTemplateMixin.html#jdaviz.core.template_mixin.PluginTemplateMixin

Reply:
added, good catch!